install/helm/gloo/templates/_helpers.tpl

```diff
@@ -31,6 +31,7 @@ Construct a container image name from a registry, repository, tag, and digest.
 {{- define "gloo.image" -}}
 {{- $image := printf "%s/%s" .registry .repository -}}
 
+{{- /* This has been copied over to _gg-helpers.tpl and should be kept in sync */ -}}
```

Comment:
why cant we have a shared space for the template to avoid sync issues?

Reply:
Yeah I was thinking the same thing. I've split the "gloo.image" template into subtemplates to handle each section